src/include/kernel/tokens.h

```diff
@@ -55,6 +55,12 @@
 #define SOF_TKN_SRC_RATE_IN			300
 #define SOF_TKN_SRC_RATE_OUT			301
 
+/* ASRC */
+#define SOF_TKN_ASRC_RATE_IN			302
```

Comment:
it'd make more sense to have your own space for ASRC, this limits the expansion for SRC, if we ever want to use additional parameters.

Reply:
Yes, I've now changed it to start from 320 (and in topology m4 too).